Source/NSMenuView.m

```diff
@@ -791,6 +791,9 @@ - (void) sizeToFit
           NSMenuItemCell *aCell = [self menuItemCellForItemAtIndex: i];
           float titleWidth = [aCell titleWidth];
```

Comment:
This should be an CGFloat and that type should also be used in your new method.

Reply:
Got it, will fix.